ceph-perf-pull-requests/config/definitions/ceph-perf-pull-requests.yml

```diff
@@ -56,9 +56,21 @@
           fi
           export NPROC=$(nproc)
           export FOR_MAKE_CHECK=true
+          cxx_compiler=g++
+          c_compiler=gcc
+          for i in $(seq 11 -1 8); do
```

Comment:
We're doing auto-detection of available Clang's version with a fallback to GCC.

Reply:
yes, starts from 11 for future-proof.